coralproject/cay
# Adding Button to Duplicate Form

## What does this PR do?

Adds functionality to copy forms.
## How do I test this PR?
- Go to View Forms
- Click copy button
- Loading spinner should replace copy button
- A new closed form with “(Copy)” appended to the title should be created, and you should be taken to its edit screen.
## Notes:

When working on this PR I noted long delays (over a second) in the action cycle. This appeared before any changes to the code, and warrants further investigation. It may be that we’re sending too much form data into the view form page…
